packages/plugin-react-refresh/index.js

```diff
@@ -116,7 +116,8 @@ function reactRefreshPlugin(opts) {
         plugins: [
           require('@babel/plugin-transform-react-jsx-self'),
           require('@babel/plugin-transform-react-jsx-source'),
-          [require('react-refresh/babel'), { skipEnvCheck: true }]
+          [require('react-refresh/babel'), { skipEnvCheck: true }],
+          ...(opts ? opts.plugins : [])
```

Comment:
When `opts.plugins` is unspecified then it will crash. Maybe we could use:
`...((opts && opts.plugins) ? opts.plugins : [])`